lib/modules/datasource/rubygems/index.ts

```diff
@@ -114,9 +114,8 @@ export class RubyGemsDatasource extends Datasource {
     const query = getQueryString({ gems: packageName });
     const url = `${path}?${query}`;
     const bufPromise = this.http.getBuffer(url);
-    return Result.wrap(bufPromise).transform(({ body }) => {
-      const data = Marshal.parse(body);
-      return MarshalledVersionInfo.safeParse(data);
-    });
+    return Result.wrap(bufPromise)
+      .transform(({ body }) => Marshal.parse(body) as NonNullable<unknown>)
```

Comment:
why do we need the cast here? 🤔

Reply:
It was workaround for `unknown` returned by `Marshal.parse`, but yeah, let's not do that